src/modules/flow/persistence/persistence.c

```diff
@@ -40,8 +40,293 @@
 #include "sol-str-table.h"
 #include "sol-util.h"
 
-#include "fs-storage.h"
-#include "efivarfs-storage.h"
+#ifdef USE_FILESYSTEM
+#include "sol-fs-storage.h"
+#endif
+
+#ifdef USE_EFIVARS
+#include "sol-efivarfs-storage.h"
+#endif
+
+#ifdef USE_MEMMAP
+#include "sol-memmap-storage.h"
+#endif
+
+static bool once = true;
+
+#ifdef USE_FILESYSTEM
+static void
+test_fs(void)
```

Comment:
I'm working on tests right now. I believe that filesystem and memmap shall be possible (EFIVARS needs some luck). Now that you asked, I'm working on fbp tests, do you think it's important to have C API tests as well? Or only fbp shall be enough?